lib/linguist/languages.yml

```diff
@@ -53,6 +53,14 @@ ABAP:
   - ".abap"
   ace_mode: abap
   language_id: 1
+ABAP CDS:
+  type: programming
+  color: "#229fc5"
+  extensions:
+    - ".asddls"
+    - ".asdcls"
+    - ".asddlxs"
```

Comment:
As per the contribution guidelines, the two last extensions don't meet our requirement on in-the-wild usage and should be removed. I haven't checked the first extension in details yet.

Reply:
All the extensions represent the same language, it's just that the entities in `asdcls` and `asddlxs` files are not as common. It would be strange to only recognize part of the language.

Reply:
as per later discussion, only `asddls` will be highlighted for now